src/pages/fundamentals.mdx

```diff
@@ -88,6 +88,10 @@ Many newcomers to React think a performant React app means simply cutting down t
 - <Resource url='https://alexsidorenko.com/blog/react-render-children-prop/'>React recursively re-renders child components, but there is a nuance</Resource>
   - Props changing will trigger a re-render or all child components directly used in a parent
   - the `children` prop changes will NOT trigger re-renders (assuming you just pass the `{children}` right through to render in the component's definition)
+- <Resource url='https://julesblom.com/writing/react-hook-component-timeline'>React component lifetime (with hooks)Timeline of a React Component With Hooks</Resource>
```

Comment:
Can we simplify this text from "React component lifetime (with hooks)Timeline of a React Component With Hooks" to be just "Timeline of a React Component With Hooks".

Using the title of the article, plus the good description/summary of the page you wrote is a great addition, I think.

Thanks again for opening this PR!

Reply:
Done!